### PR TITLE
Add export-ignore files to .gitattributes

### PR DESCRIPTION
**Pull request by ankurk91**
Hi,

Let's reduce the package weight be excluding unwanted files and folder from git archive.

Please verify that all these files and folders are not required by this package to work on consumer machine.

Thanks.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -4,3 +4,14 @@
 
 styles/* linguist-vendored
 test/**/*.txt linguist-vendored
+
+/.editorconfig        export-ignore
+/.gitattributes       export-ignore
+/.github              export-ignore
+/.gitignore           export-ignore
+/.php_cs.dist         export-ignore
+/.travis              export-ignore
```
Comment (allejo) [inline .gitattributes:13]: Will this cover both the `.travis` folder and `.travis.yml` file?

Comment (ankurk91) [reply .gitattributes:13]: @allejo 
I am not sure about it, but we can add a separate line for `.travis.yml`

Comment (allejo) [reply .gitattributes:13]: Yea if you could please have it include the folder and file, that'd be awesome!

**Comment by allejo** [comment]
Thanks for this! I didn't even know that `export-ignore` existed for `.gitattributes`

**Pull request merged**